device/nrf51822.go

```diff
@@ -0,0 +1,103 @@
+package device
+
+import (
+	"log"
+
+	"github.com/josephroberts/edge-node-manager/radio/bluetooth"
+)
+
+type operation byte
+
+/*
+Operations defined by device
+https://infocenter.nordicsemi.com/index.jsp?topic=%2Fcom.nordic.infocenter.sdk5.v11.0.0%2Fbledfu_transport_bleservice.html&anchor=ota_spec_control_state
+*/
+const (
+	START_DFU                   operation = 0x01
+	INITIALISE_DFU                        = 0x02
+	RECEIVE_FIRMWARE_IMAGE                = 0x03
+	VALIDATE_FIRMWARE_IMAGE               = 0x04
+	ACTIVATE_FIRMWARE_AND_RESET           = 0x05
+	SYSTEM_RESET                          = 0x06
+	REPORT_RECEIVED_IMG_SIZE              = 0x07
+	PKT_RCPT_NOTIF_REQ                    = 0x08
+	RESPONSE                              = 0x10
+	PKT_RCPT_NOTIF                        = 0x11
+)
+
+type response byte
+
+/*
+Responses defined by device
+https://infocenter.nordicsemi.com/index.jsp?topic=%2Fcom.nordic.infocenter.sdk5.v11.0.0%2Fbledfu_transport_bleservice.html&anchor=ota_spec_control_state
+*/
+const (
+	SUCCESS          response = 0x01
+	INVALID_STATE             = 0x02
+	NOT_SUPPORTED             = 0x03
+	DATA_SIZE                 = 0x04
+	CRC_ERROR                 = 0x05
+	OPERATION_FAILED          = 0x06
+)
+
+type updateType byte
```

Comment:
Again I don't understand why this is unexported. I guess it's used in the `device` package only, but surely it'd make more sense to export it?